agent/config/types.go

```diff
@@ -159,6 +159,9 @@ type Config struct {
 	// defined EC2 networks
 	TaskENIEnabled bool
 
+	// AppMeshEnabled specifies if the Agent is capable of invoking app mesh through CNI plugin
+	AppMeshEnabled bool
```

Comment:
Why we need this in the agent config? Should not that new agent capability for app mesh is enough?